tests/integration/utils.py

```diff
@@ -215,32 +215,45 @@ def __check_attributes_schema(actual_attributes, expected_attributes):
 
 
 # Check url for correct base and endpoint, parameters
-def check_url(self, actual_url, endpoint, query_params=None):
-    if query_params is None:
-        query_params = {}
+def check_url(self, link_url, endpoint, query_params=None):
+    query_params = {} if query_params is None else query_params
 
     base_url = self.base_url
     if self.local_test:
-        base_url = re.sub('(:\d{4}/api)', '', self.base_url)
+        '''Local instances return self links without port and /api'''
```

Comment:
Docstring should always use `"""triple double quotes"""`: https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring